docs/src/extend/custom-processors.md

```diff
@@ -102,15 +105,19 @@ See [Specify a Processor](../use/configure/plugins#specify-a-processor) in the P
 
 ## File Extension-named Processor
 
-If a processor name starts with `.`, ESLint handles the processor as a **file extension-named processor** especially and applies the processor to the kind of files automatically. People don't need to specify the file extension-named processors in their config files.
+> TODO: i don't understand this line. is it saying that as long as the file extension named processor
+> is inclueded in the plugin, it runs?
+> Users don't need to specify the file extension-named processors in their config files.
+
+If a custom processor name starts with `.`, ESLint handles the processor as a **file extension-named processor**. ESLint applies the processor to files with that filename extension automatically. Users don't need to specify the file extension-named processors in their config files.
```

Comment:
To reviewer: I have trouble understanding this line.

Is it saying that as long as the file extension named processor is included in the plugin, it runs? and therefore users don't need to specify the file extension-named processors in their config files?

would appreciate clarification so i can adjust phrasing accordingly 😄